modules/microshift-nw-router-config-IPs.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-nw-router.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-config-ip-addresses_{context}"]
+= Configuring router IP addresses
+
+You can restrict the network traffic to the router by configuring specific IP addresses. For example:
+* Use cases where the router is reachable only on internal networks, but not on northbound public networks.
+* Use cases where the router is reachable only by northbound public networks, but not on internal networks.
+* Use cases where the router is reachable by both internal networks and northbound public networks, but on separate IP addresses.
+
+.Prerequisites
+
+* You have created a {microshift-short} `config.yaml` file.
+
+* The {oc-first} is installed.
+
+//Q: Any other prereqs?
+
+.Procedure
+
+. Update the IP addresses in the `apiServer.advertiseAddress` field in the {microshift-short} `config.yaml` as shown in the following example:
++
+.Example `config.yaml` router settings
+[source,yaml]
+----
+apiServer:
+  advertiseAddress: 10.44.0.0/32
+#....
+ingress:
+  listenAddress: # <1>
+    - "" # <2>
+  ports:
+    http: 80 # <3>
+    https: 443 # <4>
+  routeAdmissionPolicy:
+    namespaceOwnership: InterNamespaceAllowed <5>
+  status: Managed # <6>
+# ....
+----
+<1> Defaults to the entire network of the host. Valid values are IP addresses or network interface names (NICs) as configured in the host.
+<2> Valid values are a single IP address or a list of IP addresses.
```

Comment:
Maybe we can merge these into a single one? Specify that the field is a list in <1>?